### DISCUSSION TITLE
Driver issues on Windows 11 Guest

### DISCUSSION THREAD

**Discussion by datarepos76** [utmapp/UTM#6147] [Windows]
### Discussed in https://github.com/utmapp/UTM/discussions/5926

<div type='discussions-op-text'>

<sup>Originally posted by **datarepos76** November 28, 2023</sup>
Hi everybody,
I recently started having the following problem on my MacBooks under UTM. When creating new Windows machines, the device drivers are apparently no longer installed correctly. The picture in the device manager is absolutely identical before and after the installation of the Guest Tools. The screen resolution can also no longer be adjusted correctly.  The problem occurs identically on both MacBooks, see below. I can no longer pinpoint exactly when the problem started, but it seems to have come with a macOS or UTM update. Do any of you have similar problems? I did some screenshots, see attachments.
Kind regards
Mathias

![drivers 2](https://github.com/utmapp/UTM/assets/145758494/69ef9afd-1fb0-4e56-93df-2b84ff5880cb)
![drivers 3](https://github.com/utmapp/UTM/assets/145758494/357d8f1b-04ff-4058-8af4-2530b10bd8e2)
![drivers 1](https://github.com/utmapp/UTM/assets/145758494/2dd45ca8-46f7-45ba-8d6a-74328760bc3c)

</div>

**Comment by datarepos76** [comment]
Here are my system details.

MacBook Air M2 on Sonoma 14.1.1, UTM 4.4.4, Windows 11 23H2 and 22H2, Guest Tools 0.229
MacBook Pro M1 Pro on Sonoma 14.1.1, UTM 4.4.4, Windows 11 23H2 and 22H2, Guest Tools 0.229

**Comment by ibousfiha** [comment]
Having the same issue on M3 Max
Any updates?

**Comment by dazza-dev** [comment]
I have the same issue. any solution?

**Comment by britus** [comment]
Me too :(
